packages/bus-core/src/workflow/registry/workflow-registry.ts

```diff
@@ -182,7 +182,11 @@ export class WorkflowRegistry {
           try {
             let workflow: Workflow<WorkflowState>
             if (container) {
-              const workflowFromContainer = container.get(options.workflowCtor)
+              const workflowFromContainer = container.get(
+                options.workflowCtor,
+                message,
+                messageAttributes
```

Comment:
does this need to conform to the context interface?

```suggestion
                options.workflowCtor,
                {  message, messageAttributes }
```

Reply:
Resolved @adenhertog